src/enums/sizes.md

```diff
@@ -17,20 +17,8 @@ enum Foo {
     B,
 }
 
-#[repr(u32)]
-enum Bar {
-    A,  // 0
-    B = 10000,
-    C,  // 10001
-}
-
 fn main() {
     dbg_size!(Foo);
-    dbg_size!(Bar);
-    dbg_size!(bool);
-    dbg_size!(Option<bool>);
-    dbg_size!(&i32);
-    dbg_size!(Option<&i32>);
```

Comment:
These are good examples to consider I think, how about moving them to the speaker notes to remind the instructor to show them in the class?

Reply:
In my experience, there is no time to go through so many examples. But we can move them to the speaker notes for people to read offline.